ui/src/app/applications/components/utils.tsx

```diff
@@ -1118,8 +1118,8 @@ export const urlPattern = new RegExp(
     )
 );
 
-export function appQualifiedName(app: appModels.Application): string {
-    return app.metadata.namespace + '/' + app.metadata.name;
+export function appQualifiedName(app: appModels.Application, nsEnabled: boolean): string {
+    return nsEnabled ? app.metadata.namespace + '/' + app.metadata.name : app.metadata.name;
```

Comment:
A minor nit, which IMO is a bit clearer (maybe not for others though, so take this with a grain of salt)

Otherwise LGTM!

```suggestion
    return `${nsEnabled ? app.metadata.namespace + '/': ''}app.metadata.name`;
```